app/src/main/res/values/strings.xml

```diff
@@ -38,9 +38,9 @@
     <string name="title_activity_calibration_override">Calibration Override</string>
     <string name="pref_header_cloud_storage">Cloud Storage</string>
     <string name="auto_configure_title">Auto configure</string>
-    <string name="prefs_auto_config_summary">Auto configure using a barcode.</string>
-    <string name="scan_share2_barcode">Scan Share Barcode</string>
-    <string name="pref_share2_scan_barcode_summary">Or scan the barcode on the share receiver</string>
+    <string name="prefs_auto_config_summary">Auto configure using a QR code.</string>
```

Comment:
As previously mentioned I think that `barcode` is a better word because a QR code is a type of `barcode` and so it better represents the feature which can support QR or other barcode types. Changing it to QR code makes it more restrictive regarding what features can be included via the auto-configure scanning and I believe that the word barcode is widely understood to mean all types of barcodes, 1D, 2D etc.

Reply:
@jamorham I noticed that you merged https://github.com/NightscoutFoundation/xDrip/pull/2022 which changes the icon to a QR code. Given that, I'm ok with keeping the _text_ as "barcode".
At the very least, the juxtaposition clarifies to the users that WE consider the terms interchangeable. 👍

Explicitly calling it a 'barcode' scanner (with the old icon) was making people like myself assume that it explicitly _didn't_ work with QR codes. This was especially confusing because it's only used for QR codes in the current state of XDrip.